plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -2857,16 +2855,16 @@ private void doTestBucketSortedTables(SchemaTableName table)
             }
 
             HdfsContext context = new HdfsContext(session);
+            HiveConfig config = getHiveConfig();
             // verify we have enough temporary files per bucket to require multiple passes
             Location stagingPathRoot;
-            if (isTemporaryStagingDirectoryEnabled(session)) {
-                stagingPathRoot = Location.of(getTemporaryStagingDirectoryPath(session)
+            if (config.isTemporaryStagingDirectoryEnabled()) {
+                stagingPathRoot = Location.of(config.getTemporaryStagingDirectoryPath()
                         .replace("${USER}", context.getIdentity().getUser()));
```

Comment:
the replacement here isn't probably important
it's tests... we could use `temporaryStagingDirectory` directly (without getHiveConfig + isTemporaryStagingDirectoryEnabled + getTemporaryStagingDirectoryPath + replace $USER)